bims/templates/main_base.html

```diff
@@ -63,6 +85,15 @@
 
     {% block foot %} {% endblock %}
 
+    <script>
```

Comment:
plese move it to https://github.com/kartoza/django-bims/blob/develop/bims/static/js/app.js
we can keep html clean from script

Reply:
done @meomancer